tc-health-client/tmagent/tmagent.go

```diff
@@ -656,7 +656,7 @@ func (pi *ParentInfo) GetTOData(cfg *config.Cfg) error {
 		if *sv.CDNName != cfg.CDNName {
 			continue
 		}
-		if sv.Type == tc.MonitorTypeName && tc.CacheStatus(*sv.Status) == tc.CacheStatusOnline {
+		if (sv.Type == tc.MonitorTypeName || sv.Type == tc.RascalTypeName) && tc.CacheStatus(*sv.Status) == tc.CacheStatusOnline {
```

Comment:
`|| sv.Type == tc.RascalTypeName` is not needed, since #7190 revert renaming `RASCAL` to `TRAFFIC_MONITOR`.